cli_flags.go

```diff
@@ -144,6 +153,13 @@ func parseArgs() (*arguments, error) {
 	fs.BoolVar(&args.verboseMode, "verbose", false, verboseModeHelp)
 	fs.BoolVar(&args.version, "version", false, versionHelp)
 
+	fs.StringVar(&args.benchDataDir, "bench-data-dir", defaultArgBenchDataDir,
+		benchDataDirHelp)
+	fs.StringVar(&args.benchProtoDir, "bench-proto-dir", defaultArgBenchProtoDir,
+		benchProtoDirHelp)
+	fs.BoolVar(&args.benchReplay, "bench-replay", defaultArgBenchReplay,
+		benchReplayHelp)
```

Comment:
I agree that the option namespace `bench` is pretty generic (the variable names are directly related and will change if the CLI options names change).
Other namespace identifiers could be `reporter` or `bench.reporter`. What is your suggestion or preference?

Also, I think that the option names could better reflect their purpose.
Another set of suggestions (if we agree on namespace `bench.reporter`):
```
-bench.reporter-input-save=<FILE> # all input goes into one ndjson file
-bench.reporter-input-load=<FILE> # all input from FILE will be replayed
-bench.reporter-proto-save=<DIR>
```

Reply:
Anything other than `[-a-z0-9]+` for arguments seems weird to me. I know that Go uses the `.` internally in some places, i.e. `-test.v`, but I'm not a fan of it.

What do you think about the following:

```
-reporter-bench-record-inputs-to=xxx
-reporter-bench-replay-inputs-from=xxx
-reporter-bench-save-outputs-to=xxx
```

Alternatively we could also go without the trailing `-to` / `-from` to trade some clarity for shorter argument names.

Reply:
Sounds mostly good to me.
What if we drop `-bench` from the option names to reduce the length. `reporter` indicates the name/function space. And for `bench`... maybe the user has different things in mind than benchmarks!?

Reply:
Yep, that's a good idea: the `-bench` is probably just confusing anyway, because the agent itself doesn't actually do any benchmarking even if the flags are being passed.

Reply:
This code is pushed. Need to update the PR description.